hack/install.sh

```diff
@@ -238,14 +238,14 @@ do_install() {
 			;;
 
 		fedora|centos)
-			cat >/etc/yum.repos.d/docker-${repo}.repo <<-EOF
+			$sh_c "cat >/etc/yum.repos.d/docker-${repo}.repo <<-EOF
```

Comment:
@tianon do you come in bulk packages? I'd like to have some of you in my toolset for when I need to do shell scripting :smile: